src/internal/bytealg/bytealg_spec.gobra

```diff
@@ -0,0 +1,74 @@
+package bytealg
+
+// +gobra
+
+import (
+	. "verification/utils/definitions"
+	sl "gobra-libs/byteslice"
+	. "gobra-libs/bytes"
+	. "bytes/spec"
+)
+
+preserves acc(sl.Bytes(b, 0, len(b)), R50)
+ensures res >= 0
+ensures len(indices) == res
+ensures forall i int :: {i in indices} i in indices ==> 0 <= i && i < len(b)
+ensures forall i int :: {i in indices} i in indices ==> ( sl.Byte(b, 0, len(b), i) == c)
+ensures forall i int :: {i in indices} !(i in indices) ==> ( i < 0 || len(b) <= i || sl.Byte(b, 0, len(b), i) != c)
+trusted
+decreases
+func Count(b []byte, c byte) (res int , ghost indices set[int])
+
+const MaxBruteForce = 64
+
+trusted
+decreases
+func Cutover(n int) int
+
+requires 2 <= len(b) && len(b) <= MaxLen
+preserves acc(sl.Bytes(a, 0, len(a)), R45)
+preserves acc(sl.Bytes(b, 0, len(b)), R45)
+ensures 0 <= res && res + len(b) <= len(a)
```

Comment:
shouldn't the result be at least -1 instead of 0? Otherwise, the postcondition in line 33 is always trivially satisfied